src/Spryker/Zed/UuidBehavior/Persistence/Propel/Behavior/UuidBehavior.php

```diff
@@ -96,7 +96,7 @@ public function objectMethods(): string
      */
     public function modifyTable(): void
     {
-        $table = $this->getTable();
+        $table = $this->getTableOrFail();
```

Comment:
Do we need to set any constraint updates due to this change?

Reply:
This will only work with [2.0.0-beta2](https://github.com/propelorm/Propel2/releases/tag/2.0.0-beta2)

Reply:
My bad, this is internal.